src/webextension/manifest.json.tpl

```diff
@@ -16,7 +16,6 @@
   },
 
   "browser_action": {
-    "browser_style": true,
```

Comment:
Interestingly, we get warnings and it seems that setting this to "false" (instead of removing it entirely) is expected/preferred?

Output from `jpm run`:

>1510092213233	addons.xpi	WARN	Please specify whether you want browser_style or not in your options_ui options.
>1510092213877	addons.webextension.lockbox@mozilla.com	WARN	Please specify whether you want browser_style or not in your browser_action options.